deployment/gRPC_proxy.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: akri-debug-echo-discovery-daemonset
```

Comment:
is this the proxy or the DH? the name of the DaemonSet and the file are contradictory

Reply:
This is the gRPC proxy been deployed as the debug echo discovery handler. 

You are right this is confusing. This is a result of using akri default debug echo installation, will take some time to make it more clear.